data/themes/classic/style.css

```diff
@@ -349,14 +349,24 @@ lmms--gui--TrackView > QWidget {
 
 /* autoscroll, loop, stop behaviour toggle buttons */
 
-/* track background colors */
+/* track background config */
 lmms--gui--TrackContentWidget {
-	qproperty-darkerColor: qlineargradient(x1:0, y1:0, x2:0, y2:1,
-							stop:0 rgb( 50, 50, 50 ), stop:0.33 rgb( 20, 20, 20 ), stop:1 rgb( 15, 15, 15 ) );
-	qproperty-lighterColor: qlineargradient(x1:0, y1:0, x2:0, y2:1,
-							stop:0 rgb( 50, 50, 50 ), stop:0.33 rgb( 40, 40, 40 ), stop:1 rgb( 30, 30, 30 ) );
-	qproperty-gridColor: rgba( 0, 0, 0, 160 );
-	qproperty-embossColor: rgba( 140, 140, 140, 64 );
+	/* colors */
+	qproperty-darkerColor: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb( 50, 50, 50 ), stop:0.33 rgb( 20, 20, 20 ), stop:1 rgb( 15, 15, 15 ) );
+	qproperty-lighterColor: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb( 50, 50, 50 ), stop:0.33 rgb( 40, 40, 40 ), stop:1 rgb( 30, 30, 30 ) );
+	qproperty-coarseGridColor: rgba(0, 0, 0, 160);
```

Comment:
You are mixing styles in this PR. As an example: The line above is according to our convention but the two lines above that is not. Here's the part describing spaces and braces.
https://github.com/LMMS/lmms/wiki/Coding-Conventions#braces

Reply:
OK. I see you have all looked into this previously and there is no need to rant.